src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    than `security.auth_retries` times within `security.auth_delay` seconds,
+    no authentication delay is enforced. The retry counter is also reset after
+    any successful authentication attempt.
+]])
+
+M['security.auth_type'] = format_text([[
+    Specify a protocol used to authenticate users. The possible values are:
+
+    - `chap-sha1`: use the CHAP protocol with SHA-1 hashing applied to
+    passwords.
+    - `pap-sha256`: use PAP authentication with the SHA256 hashing algorithm.
+
+    Note that CHAP stores password hashes in the `_user` space unsalted. If an
+    attacker gains access to the database, they may crack a password, for
+    example, using a rainbow table. For PAP, a password is salted with a
+    user-unique salt before saving it in the database, which keeps the
+    database protected from cracking using a rainbow table.
+]])
+
+M['security.disable_guest'] = format_text([[
+    If `true`, turn off access over remote connections from unauthenticated or
+    guest users. This option affects connections between cluster members and
+    `net.box` connections.
+]])
+
+M['security.password_enforce_digits'] = format_text([[
+    If true, a password should contain digits (0-9).
+]])
+
+M['security.password_enforce_lowercase'] = format_text([[
+    If true, a password should contain lowercase letters (a-z).
+]])
+
+M['security.password_enforce_specialchars'] = format_text([[
+    If true, a password should contain at least one special character
+    (such as &|?!@$).
+]])
+
+M['security.password_enforce_uppercase'] = format_text([[
+    If true, a password should contain uppercase letters (A-Z).
+]])
+
+M['security.password_history_length'] = format_text([[
+    Specify the number of unique new user passwords before an old password
+    can be reused. Note tarantool uses the auth_history field in the
+    `box.space._user` system space to store user passwords.
+]])
+
+M['security.password_lifetime_days'] = format_text([[
+    Specify the maximum period of time (in days) a user can use the
+    same password. When this period ends, a user gets the "Password expired"
+    error on a login attempt. To restore access for such users,
+    `use box.schema.user.passwd`.
+]])
+
+M['security.password_min_length'] = format_text([[
+    Specify the minimum number of characters for a password.
+]])
+
+M['security.secure_erasing'] = format_text([[
+    If `true`, forces Tarantool to overwrite a data file a few times before
+    deletion to render recovery of a deleted file impossible. The option
+    applies to both `.xlog` and `.snap` files as well as Vinyl data files.
+]])
+
+-- }}} security configuration
+
+-- {{{ sharding configuration
+
+M['sharding'] = format_text([[
+    This section defines configuration parameters related to sharding.
+]])
+
+M['sharding.bucket_count'] = format_text([[
+    The total number of buckets in a cluster. Learn more in Bucket count.
+]])
```

Comment:
The 'learn more' reference without a link has no sense.

Reply:
Dropped.